xds/internal/resolver/xds_resolver.go

```diff
@@ -46,7 +46,8 @@ const Scheme = "xds"
 
 // newBuilderWithConfigForTesting creates a new xds resolver builder using a
 // specific xds bootstrap config, so tests can use multiple xds clients in
-// different ClientConns at the same time.
+// different ClientConns at the same time. It creates the new xds client int
```

Comment:
typo/nit: "It creates a new xds client in the default pool with the provided config."

Reply:
Done